packages/test/test-gc-sweep-tests/src/test/gcSweepTest.spec.ts

```diff
@@ -86,27 +86,27 @@ describeNoCompat("GC Sweep tests", (getTestObjectProvider) => {
     let overrideLogger: IgnoreErrorLogger;
 
     const sleep = async (ms: number) => new Promise((resolve) => setTimeout(resolve, ms));
-    // TODO: run the test with multiple seeds consistently
-    const seed = Math.random();
-    const random: IRandom = makeRandom(seed);
-
-    // Improve this to distribute the randomness among relevant GC times
-    const randomInteger = (min: number, max: number): number => {
-        return random.integer(min, max);
-    };
 
     // Time spent to run the test. Currently 10 seconds, for better coverage increase this number.
     const testTime = 10 * 1000;
 
     // Currently for GC Sweep testing only, run with npm run test. Should not be running in CI
     // Note: can run with npm run test:build to build and run the test
     // TODO: setup test to run in CI
+    const numberOfTests = 10;
```

Comment:
Could be cool to have this configurable via Mocha cmd arguments.  For a later PR.